PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
What would be the benefit of adding it as another new property? Which other scenarios are you thinking of? I cannot think of one right now, because in not standalone mode there would be most probably write rights problems and this path is less likely to be used.

Why is it returning Location? I don't know right now, I checked my notes just to be fully sure, the method is taken from the NUnit code base, which in turn was taken from other sources, but it's tested inside the NUnit test suite itself, so I think it's safe to use.

Your approach can be used for FD, but it's not compatible with all possible .NET use cases, which I think it's a good idea.  A more complete one would be:

    Uri codeBase = new Uri(cb);
    return codeBase.IsFile ? codeBase.LocalPath : codeBase.ToString();